tags.go

```diff
@@ -55,35 +51,28 @@ type TagDeleteInput struct {
 
 //#region Assign
 
+// Deprecated: Use AssignTagsFor instead
 func (client *Client) AssignTagsForAlias(alias string, tags map[string]string) ([]Tag, error) {
-	input := TagAssignInput{
-		Alias: alias,
-		Tags:  []TagInput{},
-	}
-	for key, value := range tags {
-		input.Tags = append(input.Tags, TagInput{
-			Key:   key,
-			Value: value,
-		})
-	}
-	return client.AssignTags(input)
+	return client.AssignTags(alias, tags)
 }
 
+// Deprecated: Use AssignTagFor instead
 func (client *Client) AssignTagForAlias(alias string, key string, value string) ([]Tag, error) {
-	input := TagAssignInput{
-		Alias: alias,
-		Tags:  []TagInput{},
-	}
-	input.Tags = append(input.Tags, TagInput{
-		Key:   key,
-		Value: value,
-	})
-	return client.AssignTags(input)
+	return client.AssignTags(alias, map[string]string{key: value})
 }
 
+// Deprecated: Use AssignTagsFor instead
```

Comment:
The functions around tags where VERY messy - so i DRY'd them up and deprecated what we could.  I'm still not 100% happy but its a step in the right direction for increasing test coverage